opensearch-operator/pkg/helpers/helpers.go

```diff
@@ -359,3 +360,28 @@ func CompareVersions(v1 string, v2 string) bool {
 	ver2, _ := version.NewVersion(v2)
 	return err == nil && ver1.LessThan(ver2)
 }
+
+func GetJvmHeapSize(nodePool *opsterv1.NodePool) string {
```

Comment:
```suggestion
func CalculateJvmHeapSize(nodePool *opsterv1.NodePool) string {
```
Get would imply it does not do any logic.

Reply:
Yes, it makes sense. I changed the name of the function to `CalculateJvmHeapSize`.